react/src/lib/components/DeckGLMap/layers/terrain/terrainMapLayer.ts

```diff
@@ -65,6 +65,9 @@ export interface TerrainMapLayerProps<D> extends SimpleMeshLayerProps<D> {
 
     // Use color map in this range.
     colorMapRange: [number, number];
+
+    //If true readout will be z value (depth). Otherwise it is the texture property value.
+    isReadoutDepth: boolean;
 }
 
 const defaultProps = {
```

Comment:
We should have created default  props in layersDefaultProps.ts
May be in another review.